Vagrantfile

```diff
@@ -201,7 +201,7 @@ Vagrant.configure(2) do |config|
     # Che is active once it returns success        
     while [ true ]; do
       printf "#"
-      curl -v ${CHE_URL}/dashboard &>/dev/null
```

Comment:
It won't show if API is running. I think you have to parse respond code to be equal to 200. Trailing slash is also needed.